home-assignments/session1/exercise1.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import json
+import yaml
+
+args = sys.argv
+if len(args) < 2:
+    print('Error: missing file name')
+    exit(1)
+
+infile = args[1]
+fp = open(infile)
+j = json.load(fp)
+fp.close()
+
+borders = sorted(j['buckets'])
+buckets = {border:[] for border in borders}
+other = []
```

Comment:
It was said in the exercise, that you should add bucket dynamically, if you need it